src/soap/admin.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib import admin
+from django.utils.translation import gettext_lazy as _
+
+from soap.models import SoapService
```

Comment:
```suggestion
from .models import SoapService
```
let's use relative imports inside the same app, in case it needs to be moved around more